new-frontiers-14/frontier-station-14
# Port gavel from Delta-V (#660), better powdered wig, courthouse revamp

## About the PR



Adds the gavel from [Delta-V#660](https://github.com/DeltaV-Station/Delta-v/pull/660), along with the gavel block, trial timer, and a separate audio set.

Edits the powdered wig sprite to have a back when viewed facing north.

Touched up the Courthouse:
- Added powdered wig, judge's robe, gavel and gavel block to the table
- Added a trial timer to the courthouse
  - Sin: currently mapped on a window block just so the rest of the court is visible and the thing isn't floating in the air.  Sucks, but A: we need a timer, and B: you should be able to see the rest of the courthouse.
- Moved the camera down by a square, rerouted the gas piping beneath it.
- Moved the FuelVend machines in the middle of the docks to the corners, moved them out by a square.
- Made the giant, 7-square long benches into two 3-square long benches with potted plants in between.
- Timer in the brig swapped to a security-access locked brig timer.

## Why / Balance


The gavel's been talked about for ages, and it's a nice touch, the trial timer looks better than the black metal ones.
The powdered wig always bugged me.
The Courthouse changes should give better QoL for whenever that place is used.

## How to test


1. Visit the courthouse, bang the gavel.
2. View the rest of the changes listed above, see if I didn't mess anything up.

## Media


Changes to the courthouse.  Judge's attire laid out on the table, timer located on the reinforced glass block.  Camera and scrubber moved down a tile.
![courthouse](https://github.com/user-attachments/assets/8a8d5d14-d5a6-4e9b-8055-051663042d1e)

West dock, note the FuelVend in the corner.
![west_dock](https://github.com/user-attachments/assets/0e8b8d90-5bac-4d6f-88cf-e1e253c8adcb)

East dock, note the FuelVend in the corner.
![east-dock](https://github.com/user-attachments/assets/72557c25-9142-4c00-b028-ae65a564f926)

Courtyard, note the smaller benches.
![courtyard](https://github.com/user-attachments/assets/a861c587-a844-41c4-bc17-a1ee951f0cec)

Wolfgang Amadeus Mothzart in the fluff.
![image](https://github.com/user-attachments/assets/b95d3af4-577e-4f05-8424-495500efdb5e)

## Requirements



## Breaking changes


**Changelog**


:cl: whatston3 and leonardo-dabepis
- add: Added a gavel, gavel block, powdered wig to the courthouse.
- tweak: Remodeled the Courthouse slightly.
